prime-dotnet-webapi/Services/AdminService.cs

```diff
@@ -84,6 +91,24 @@ public async Task<IEnumerable<Admin>> GetAdminsAsync()
                 .ToListAsync();
         }
 
+        public async Task<IEnumerable<AdminUserViewModel>> GetAdminUserListAsync()
+        {
+
+            return await _context.Admins.Select(a =>
```

Comment:
Should be a defined order ... question for Business?

Reply:
how about sorting by first name first? let's review when it is in UAT.